query-builder-elasticsearch.js

```diff
@@ -108,6 +110,8 @@
                         // we express this with a sub boolean query and must_not.
                         if (data.condition === 'OR' && (rule.operator === 'not_equal' || rule.operator === 'not_in')) {
                             return {'bool': {'must_not': [part]}}
+                        } else if (rule.operator === 'is_null' || rule.operator === 'is_not_null') {
```

Comment:
This said, do not hesitate to tell me if this incorrect, maybe there are different ways to translate an OR query.